Index: config.yaml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>server: 'localhost'\r\npassword: 'Hoanglinh@99'\r\ndatabaseName: 'sax'\r\nport: '1434'\r\nusername: 'sa'\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/config.yaml b/config.yaml
--- a/config.yaml	(revision ac3a05b3ec7f71457264d5f5b15d5a27d5a6e39b)
+++ b/config.yaml	(date 1701768825348)
@@ -1,5 +1,5 @@
 server: 'localhost'
-password: 'Hoanglinh@99'
+password: '06122004'
 databaseName: 'sax'
-port: '1434'
+port: '1433'
 username: 'sa'
